### ISSUE TITLE
Script's switch doesn't reflect real state

### ISSUE THREAD

**Issue by TheFax** [home-assistant/frontend#5364]
## Checklist

- [Y] I have updated to the latest available Home Assistant version.
- [Y] I have cleared the cache of my browser.
- [Y] I have tried a different browser to see if it is related to my browser.

## The problem

I have a **script** in my hassio that:
1. turns on a fan
2. wait some time (**30...60 minutes**, depends by a slider)
3. then turn off the fan.

This is the source of the script:
```
forced_ventilation:
  alias: "Ventilazione a tempo"
  sequence:
    - data:
        entity_id: switch.fan
      service: switch.turn_on
    - delay:
        minutes: "{{ states('input_number.slider_fan') | int }}"
    - data:
        entity_id: switch.fan
      service: switch.turn_off
```
The only way to start the script is by a Switch on the Lovelace interface.

The probelm is when I turn on the switch, **it returns to OFF position after some seconds**, but the script is not yet terminated and the fan is active. **If I update the Home Assistant webpage (pressing F5 for example), the switch returns to ON position as expected**.

Important: the script works regularly, and the fan is turned on and off with the right delay.

![example](https://user-images.githubusercontent.com/19174767/77888876-371ef980-726d-11ea-9ed0-5507873b82e7.gif)

## Expected behavior

When I turn on the switch, it should remain in ON position until the script is terminated (more tan 30 minutes, for my script). 
When I turn off the switch, it should stop the script execution and go to the OFF position and remain in OFF position.

## Steps to reproduce

1. Create a script with the source code I have pasted above
2. Create a switch that execute this script on the lovelace interface
3. Click on the switch using your web browser.
4. The switch goes to ON position. The fan starts. **But after some seconds it returns to OFF position.** The fan is still working as expected.
5. Refresh the webpage (for example, pressing F5). **The switch goes to ON position.**

(optional)
6. Turn off the switch.
7. The switch goes to OFF position. **But after some seconds the switch returns to ON position.**
8. Refresh the webpage. **The switch goes to OFF position"."

## Environment

- Home Assistant release with the issue: **Hassio 107, HassOS 3.12, Supervisor 213, Standard Lovelace interface.**
- Last working Home Assistant release (if known): **I use Hassio since 0.94, and this issue has been always present**
- Browser and browser version: **Tested with Firefox and Chrome, in both smarphone and PC, with both LAN or over internet connection.**
- Operating system: **Windows 10, Android**
- Fan connected via Mosquitto broker 5.1

## Javascript errors shown in your browser console/inspector
It seems to me that there is no problems on the console/inspector

**Comment by iantrich** [comment]
duplicate of #3276

**Comment by TheFax** [comment]
**Solved** with **Home Assistant 0.113**
